services/database-analyzer-sequelize.js

```diff
@@ -0,0 +1,171 @@
+const ColumnTypeGetter = require('./column-type-getter');
+const TableForeignKeysAnalyzer = require('./table-foreign-keys-analyzer');
+const _ = require('lodash');
+const logger = require('./logger');
+const P = require('bluebird');
```

Comment:
[Conventions?] For readability I would move dependency require on top and internal code after that.
``` javascript
const P = require('bluebird');
const _ = require('lodash');
const logger = require('./logger');
const ColumnTypeGetter = require('./column-type-getter');
const TableForeignKeysAnalyzer = require('./table-foreign-keys-analyzer');
const { DatabaseAnalyzerError } = require('../utils/errors');
```

Reply:
Totally agree. Thanks & sorry